hal/halx86/apic/apictimer.c

```diff
@@ -63,22 +69,75 @@ VOID
 NTAPI
 HalStartProfileInterrupt(IN KPROFILE_SOURCE ProfileSource)
 {
-    UNIMPLEMENTED;
-    return;
+	LVT_REGISTER LvtEntry;
+
+	/* Only handle ProfileTime */
+	if (ProfileSource == ProfileTime)
```

Comment:
The PIC version seems to just ignore the parameter. Do you have a specific reason to check for it? If not, just doing UNREFERENCED_PARAMETER seems preferable.

Reply:
> Do you have a specific reason to check for it?

Yes I am. Win2003 SP2 x64 kernel only handles ProfileSource 0 (which is ProfileTime).